shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -494,19 +492,22 @@ instance Era era => FromCBOR (PPUPState era) where
       fppup <- fromCBOR
       pure $ PPUPState ppup fppup
 
-pvCanFollow :: ProtVer -> StrictMaybe ProtVer -> Bool
-pvCanFollow _ SNothing = True
-pvCanFollow (ProtVer m n) (SJust (ProtVer m' n')) =
+pvCanFollow :: ProtVer -> ProtVer -> Bool
+pvCanFollow (ProtVer m n) (ProtVer m' n') =
```

Comment:
the version of `pvCanFollow` that has a Maybe as a second parameter is very tailored to it's one use case in another module, namely validating a protocol parameter update. since I am now adding a second validation on the update votes I thought I would make a general function (see `getInvalidValue` in the PPUP rule) that hunts through an update proposal with a particular value and predicate and returns a counterexample if it finds one. and then I could re-use this for my new `d` validation.

Reply:
mute now anyway since I've just removed the `d` validation.